doc/articles/uno-development/building-uno-ui.md

```diff
@@ -135,7 +135,7 @@ The tool needs to be run on Windows because of its dependency on the Windows SDK
 To run the synchronization tool:
 
 - Open a `Developer Command Prompt for Visual Studio` (2019 or 2022)
-- Go the the `uno\build` folder (not the `uno\src\build` folder)
+- Go the `uno\build` folder (not the `uno\src\build` folder)
```

Comment:
```suggestion
- Go to the `uno\build` folder (not the `uno\src\build` folder)
```

Reply:
Change will be done on this follow up PR: https://github.com/unoplatform/uno/pull/15144